test/models/mdetr/test_text_encoder.py

```diff
@@ -41,7 +39,7 @@ def setUp(self):
             normalize_before=False,
         )
 
-        self.text_encoder = MDETRTextEncoder(
+        self.text_encoder = TextEncoder(
```

Comment:
Should we remove `test_mdetr_text_embeddings` and `test_mdetr_text_encoder` since both should have been covered by unit tests of `BERTTextEmbeddings` and `(BERT)TextEncoder`? No need to spend CI resource if we are testing the same codepaths.

cc @ebsmothers

Reply:
Yeah I agree, I think it should be fine to remove these now